server/react-admin/src/videoTestimonials/VideoTestimonialForm.js

```diff
@@ -7,9 +7,10 @@ import Paper from '@mui/material/Paper'
 import Snackbar from '@mui/material/Snackbar'
 import Typography from '@mui/material/Typography'
 import TextFieldInput from '../form/TextFieldInput'
+import CheckboxInput from '../form/CheckboxInput'
 
 // eslint-disable-next-line camelcase
-const emptyFormValues = { subject: '', comment: '', video_cms_id: '' }
+const emptyFormValues = { subject: '', comment: '', video_cms_id: '', prvcy_stmt_ack: false }
```

Comment:
I think the attribute is `privacy_stmt_ack` but even after fixing I can't get the "Add" to work.

Reply:
Yea I'm still working through this, updates and adds don't go through. I'm trying to get the value to change on the form

Reply:
I think you want to avoid tracking the state of the thing yourself. (delete the `checked` part of the state. The tricky part here is that when expanding the `field` object it doesn't have `checked` so we have to set that ourselves. 

`	<Controller
          name={name}
	 control={control}
	 defaultValue={defaultValue || false}
	 rules={rules}
	 render={({field}) => (
            <Checkbox checked={field.value} {...field} />
          )}
       />
`